app/javascript/components/patient/laboratory/LaboratoryModal.js

```diff
@@ -120,6 +130,30 @@ class LaboratoryModal extends React.Component {
                 </Form.Control>
               </Form.Group>
             </Row>
+            {this.props.editMode &&
+              this.props.isolation &&
+              this.props.onlyPosLab &&
+              (this.state.result !== 'positive' || this.state.specimen_collection == null) && (
+                <div>
+                  <p className="mt-2 mb-3">
```

Comment:
might be worth calling out this text more, maybe bolding it? or putting it in a warning block?

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/files#diff-a843ed07f9133d7fd95ab2bda4a43af2ad74f1be080aeddb353793998d85c5ffR138